nexus/pkg/env/artifacts.go

```diff
@@ -0,0 +1,22 @@
+package env
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func GetArtifactDir() (artifactDir string, rerr error) {
```

Comment:
Nit: we probably don't need to bother with `Getwd()` and `Abs()`, the Python implementation doesn't ([code](https://github.com/wandb/wandb/blob/4f6e172f42361253d9acde2c119d9006da0ec6b9/wandb/env.py#L380)).